cypress/integration/bcp47Tag.spec.ts

```diff
@@ -0,0 +1,25 @@
+describe("BCP47Tag component test cases", function () {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should not show error message when valid tag is given", function () {
+    cy.contains("label", "Enter BCP 47 tag");
+    cy.get("#tag-input").type("zh");
+    cy.contains("label", "Enter BCP 47 tag");
+  });
+
+  it("should show error message when invalid tag is given", function () {
+    cy.contains("label", "Enter BCP 47 tag");
+    cy.get("#tag-input").type("012345");
+    cy.contains("label", "Invalid tag");
+  });
+
+  it("should clears error message when valid tag is given", function () {
+    cy.contains("label", "Enter BCP 47 tag");
+    cy.get("#tag-input").type("012345");
+    cy.contains("label", "Invalid tag");
+    cy.get("#tag-input").clear();
```

Comment:
Not necessary for this PR, but for future reuable components, it's better to target the [`data-cy` attribute](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements) when testing with Cypress.

```suggestion
    cy.get("[data-cy=tag-input]"").type("zh");
    cy.contains("label", "Enter BCP 47 tag");
  });

  it("should show error message when invalid tag is given", function () {
    cy.contains("label", "Enter BCP 47 tag");
    cy.get("[data-cy=tag-input]").type("012345");
    cy.contains("label", "Invalid tag");
  });

  it("should clears error message when valid tag is given", function () {
    cy.contains("label", "Enter BCP 47 tag");
    cy.get("[data-cy=tag-input]").type("012345");
    cy.contains("label", "Invalid tag");
    cy.get("#tag-input").clear();
```